multipaz-zkp/src/commonMain/kotlin/org/multipaz/zkp/exception/CircuitGenerationException.kt

```diff
@@ -0,0 +1,4 @@
+package org.multipaz.zkp.exception
```

Comment:
We generally don't put exceptions in their own package, suggest to just use the `org.multipaz.zkp` namespace.